sdk/storage/Azure.Storage.Blobs/src/Azure.Storage.Blobs.csproj

```diff
@@ -1,4 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
```

Comment:
VS doesn't like projects without TargetFrameworks it treats them as old-style projects and tries to migrate them.

Reply:
SO THAT'S WHY IT KEPT DOING THAT?!?!??!

Thanks.

This is a very good change.

Reply:
Took me a while to figure this out

Reply:
Yeah that is why I originally had them in the project file directly but I was hoping VS2019 was better about this but I guess not :(

To avoid confusion can you also remove https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/storage/Directory.Build.props#L28. As well as add this back to the template test project that I accidentally removed https://github.com/Azure/azure-sdk-for-net/pull/6247/files#diff-2e2cf9bbf29798332604d9960da39caaL1

Thanks